src/main/com/example012/v1/Tickets.java

```diff
@@ -0,0 +1,93 @@
+package main.com.example012.v1;
+
+import java.security.NoSuchAlgorithmException;
+import java.security.SecureRandom;
+import java.util.Arrays;
+import java.util.UUID;
+
+public class Tickets {
+
+    private int[] lotteryTicketNumbers;
+
+    private UUID id;
+
+    private Company gameType;
+
+    private int correctNumbers;
+
+    public Tickets tickets(Company gameType) {
+
+        this.gameType = gameType;
+        setLotteryTicketNumbers();
+        setId();
+        return this;
```

Comment:
return this ne ... budeš muset vyřešit nějak elegantněji ... ale to vyplyne z toho, jak to budes refaktorovat podle komentaru